src/ansys/templates/python/pyace_pckg/{{cookiecutter.__project_name_slug}}/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# {{ cookiecutter.project_name }} {{ cookiecutter.library_name }} CHANGELOG
```

Comment:
Could we please add this file to `templates/python/common/{{cookiecutter.__project_name_slug}}`? This will ensure all Python templates to have a `CHANGELOG.md`. Thanks for adding this, @asbfall 🚀

Reply:
@jorgepiloto  Yes I do agree with your suggestion (100% 👍 ). For the time being I did not wanted to touch anything under the common folder before we discuss it together.